torch/csrc/jit/runtime/argument_spec.h

```diff
@@ -241,7 +241,7 @@ struct CompleteArgumentInfo;
 struct CompleteArgumentSpec {
   CompleteArgumentSpec(bool with_grad, at::ArrayRef<IValue> inputs)
       : ninputs(inputs.size()) {
-    int32_t all_dims = 0;
+    int64_t all_dims = 0;
```

Comment:
Eh, not sure this is needed since I don't think we can have int64 dims anytime soon, but probably harmless.